candi-meter/candi-meter-ui/package.json

```diff
@@ -15,9 +15,12 @@
     "@angular/compiler": "^14.0.0",
     "@angular/core": "^14.0.0",
     "@angular/forms": "^14.0.0",
+    "@angular/material": "^7.0.0",
```

Comment:
why do you use an old version from angular material
please convert to the version applicable with angular version 14